src/components/SearchResult/SearchResult.jsx

```diff
@@ -13,30 +11,13 @@ function SearchResults({ query, results }) {
           aria-live="assertive"
         >
           {`Found ${results.length} `}
-          {results.length === 1 && "module matching with"}
-          {results.length > 1 && "modules matching with"}
+          {results.length === 1 && `${type} matching with`}
+          {results.length > 1 && `${type}s matching with`}
           {` "${query}"`}
         </h4>
       )}
       <h4>{results.length < 1 && "Found nothing"}</h4>
-      {!!results.length && (
-        <Row className="justify-content-center">
-          {results.map(
-            ({ title, path, cover, tags, excerpt, date, author }) => (
-              <Cards
-                title={title}
-                path={path}
-                cover={cover.childImageSharp}
-                tags={tags}
-                excerpt={excerpt}
-                date={moment(date).format("MMMM DD, YYYY")}
-                author={author}
-                  type={type}
-              />
-            )
-          )}
-        </Row>
-      )}
+      <PostListing prefix={`/${type}`} postList={results} />
```

Comment:
Probably the `SearchResult` should receive the `prefix` in addition to the `type`, they don't need to be the same (they currently just happen to be the same)